src/qt/pivx/settings/settingswidget.cpp

```diff
@@ -157,6 +157,15 @@ SettingsWidget::SettingsWidget(PIVXGUI* parent) :
     connect(settingsWalletOptionsWidget, &SettingsWalletOptionsWidget::message, this, &SettingsWidget::message);
     connect(settingsInformationWidget, &SettingsInformationWidget::message,this, &SettingsWidget::message);
 
+    connect(settingsDisplayOptionsWidget, &SettingsDisplayOptionsWidget::saveSettings, this, &SettingsWidget::onSaveOptionsClicked);
+    connect(settingsDisplayOptionsWidget, &SettingsDisplayOptionsWidget::discardSettings, this, &SettingsWidget::onDiscardChanges);
```

Comment:
We can do this simpler without have to repeat the `saveSettings` and `discardSettings` code on each settings options object.

The `SettingsWidget` parent is passed in every child constructor to the object as the `QWidget* parent`, they could just use it  and connect directly to `onSaveOptionsClicked` and `onDiscardChanges` as it was before..

The other option would be create a base class for the settings option object but seems to be little bit too much for this.